src/content/docs/infrastructure/host-integrations/host-integrations-list/microsoft-sql/microsoft-sql-server-integration.mdx

```diff
@@ -200,6 +204,44 @@ Create a new login and grant the necessary permissions:
     ```
 
   </Collapser>
+
+  <Collapser
+    id="domain-user"
+    title="For Windows Group Managed Service Account"
+  >
+
+
+    1. Create a new login and grant `CONNECT` and `VIEW` permissions:
+
+    ```sql
+    USE master;
+      CREATE LOGIN [DomainName\gMSA] FROM windows;
+      GRANT CONNECT SQL TO [Domain\gMSA];
+      GRANT VIEW SERVER STATE TO [Domain\gMSA];
+      GRANT VIEW ANY DEFINITION TO [Domain\gMSA];
+    ```
+
+    2. Use the following statements to grant `READ` access privileges to the user:
```

Comment:
Where are they using the following statements? Is it immediately after the snippet that grants CONNECT and VIEW permissions? Is it later in some kind of file? What file are these updates located in?

Reply:
This is just the setup of the gMSA on the SQL Server side. I was just following the pattern for creating the domain accounts or the SQL login account.